images/recycler/scripts/recycler.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# 'recycler' performs an 'rm -rf' on a volume to scrub it clean before it's
+# reused as a cluster resource. This script is intended to be used in a pod that
+# performs the scrub. The container in the pod should succeed or fail based on
+# the exit status of this script.
+
+set -e -o pipefail
+shopt -s dotglob nullglob
```

Comment:
Can we please use 
```
set -o errexit
set -o noglob
set -o nounset
set -o pipefail
```
and get rid of the `shopt` entirely since we aren't using `*` anymore?